include/aikido/planner/World.hpp

```diff
@@ -21,7 +21,7 @@ class World
   // Encapsulates the state of the World.
   struct State
   {
-    std::unordered_map<std::string, dart::dynamics::Skeleton::Configuration>
+    std::unordered_map<std::string, ::dart::dynamics::Skeleton::Configuration>
```

Comment:
Just wondering, what is the conflicting namespace that requires pre-pended colons here? (As I understand it, this is chiefly useful if the class name exists in the global namespace and a sub-namespace that you're in. But I don't think either the aikido or planner namespaces have conflicting class names with this.)

Reply:
The namespace aikido::planner::dart exists (for planners that rely on DART body nodes, e.g. planToTSR). Honestly, I think it needs to be reorganized, but that's a discussion for a different PR.

I got a compile error when including both World and a planner::dart header file in Robot.hpp saying that `dynamics` isn't a namespace in `aikido:: planner:: dart`, so here we are.